### PR TITLE
Recover password screen e2e tests

### PR DESCRIPTION
**Pull request by galeza**
Contains tests for Recovery Password screen:
-FP logo is visible and clickable
-Recover Password page contains heading and image
-Recover Password button is disabled when screen is open
-Leaving email field blank triggers error and Recover Password button is disabled
-Entering invalid email triggers error and Recover Password button is disabled
-Entering correct email triggers Recovery Button is enabled
-Back to Sign in screen link is visible and clickable

I added custom commands and move some code there. We have the same code verifying logo and page tile/image. So using custom command can help us to keep tests DRY. We could refactor other tests to avoid code repetitions.

I check if button Recover Password is disable when user interacting with email field by entering incorrect email or leave this field empty.

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

**Review by sheeba1 — COMMENTED** [review]
@galeza , Noticed the label 'Your Email'  is not being tested to see if its present .Thanks!

**Review by jessica-lau — APPROVED** [review]
LGTM!

**Review by idjevm — APPROVED** [review]
LGTM! Let me know if you guys want me to merge this one or wait for more reviews!

**Review by sheeba1 — APPROVED** [review]
@galeza , Noticed the label 'Your Email'  is not tested to see if its present .Thanks!

**Comment by galeza** [comment]
@idjevm 
Please merge.

**Comment by idjevm** [comment]
Actually we will have to wait on some stuff. Looks like there is a code freeze by @joshmorel, I think he posted in the engineering channel that we shouldn't merge yet

**Comment by galeza** [comment]
@idjevm 
ok!

**Review by idjevm — APPROVED** [review]
Code freeze is over! Merging this now! Thanks Agnieszka

**Pull request merged**
